openmetadata-ui/src/main/resources/ui/src/components/AddIngestion/Steps/ConfigureIngestion.tsx

```diff
@@ -112,6 +114,47 @@ const ConfigureIngestion = ({
     );
   };
 
+  const [profileSampleError, setProfileSampleError] = useState({
+    profileSample: false,
+  });
+
+  const handleProfileSampleValidation = (profileSampleValue: number) => {
+    const errMsg = cloneDeep(profileSampleError);
```

Comment:
Not sure why we need `object` here direct boolean will work. Am I missing something here?